content/200-concepts/300-more/400-comparisons/01-prisma-and-typeorm.mdx

```diff
@@ -150,19 +150,6 @@ const posts = await postRepository.find({
 
 TypeORM only offers limit-offset pagination while Prisma conveniently provides dedicated APIs for both limit-offset but also cursor-based. You can learn more about both approaches in the [Pagination](../../components/prisma-client/pagination) section of the docs or in the API comparison [below](#pagination-1).
 
-### Transactions
```

Comment:
I was debating whether or not to delete or rewrite this section.

There are still a few minor differences: they support isolation levels, we support more ways to do transactions. Decided it wasn't worth calling out as differences, but happy to revisit if you feel differently.